client/src/document/organisms/metadata/index.scss

```diff
@@ -0,0 +1,22 @@
+@import "~@mdn/minimalist/sass/vars/color-palette";
+@import "~@mdn/minimalist/sass/vars/layout";
+@import "~@mdn/minimalist/sass/vars/typography";
+
+.metadata {
+  background-color: $neutral-550;
+}
+
+.metadata-content-container {
+  margin: 0 auto;
+  max-width: $max-width-default;
+  padding: $base-spacing;
+
+  .on-github {
+    margin-bottom: $base-spacing;
+  }
+
+  .last-modified-date {
+    font-size: $tiny-text;
```

Comment:
I don't like this. It doesn't make sense to me. 
Here's what it would look like with a smaller font:
<img width="855" alt="Screen Shot 2020-12-08 at 10 22 31 AM" src="https://user-images.githubusercontent.com/26739/101502851-7266e180-393f-11eb-88ec-67ec03cd1afb.png">

There's no real-estate problem that requires that this generally pretty useful piece of information has to be tiny now. 
It made sense when it was closer to the content because then a smaller size would make it "stand out" so it's more obvious that it's not part of the text.

Reply:
Ah, the reason I did that is that I remember someone mentioning that that information should not be so prominant. Happy to remove.